packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -266,25 +268,34 @@ export class Tooltip extends React.Component {
 }
 
 Tooltip.defaultProps = {
+  component: 'button',
   interactiveBorder: 15,
-  placement: 'top',
   maxWidth: '300px',
-  zIndex: '9999',
   offset: [0, 5],
-  triggerComponent: 'button',
+  placement: 'top',
   transitionDuration: 250, // Equivalent to $animation-speed-1
+  zIndex: '9999',
 };
 Tooltip.propTypes = {
+  /**
+   * Classes applied to the tooltip trigger when the tooltip is active
+   */
+  activeClassName: PropTypes.string,
   /**
    * Helpful description of the tooltip for screenreaders
    */
   ariaLabel: PropTypes.string,
   /**
-   * Content inside the tooltip body or popover. If this contains interactive elements set the `interactive` prop.
+   * Tooltip trigger content
    */
   children: PropTypes.node.isRequired,
   /**
-   * Classes applied to the tooltip body
+   * When provided, this will render the passed in component for the tooltip trigger. Typically this will be a `button`, `a`,
+ or rarely an `input` element.
```

Comment:
This ends up wrapping the entire tooltip trigger, right? It might be good to spell that out a little more in this description.